c7n/resources/vpc.py

```diff
@@ -324,7 +324,7 @@ def compute_rule_hash(self, rule):
             ev.sort()
             for e in ev:
                 buf += "%s-" % e
-        return abs(zlib.crc32(buf.encode('ascii')))
+        return zlib.crc32(buf.encode('ascii')) & 0xffffffff
```

Comment:
Heh. That one is lifted straight from [the official docs](https://docs.python.org/3/library/zlib.html#zlib.crc32) and the commit message explains it, but I guess it can’t hurt to clarify in a comment.